internal/venice-test-common/src/main/java/com/linkedin/venice/utils/DataProviderUtils.java

```diff
@@ -164,6 +164,19 @@ public static Object[][] ingestionTaskReusableObjectsStrategy() {
     return allPermutationGenerator(IngestionTaskReusableObjects.Strategy.values());
   }
 
+  @DataProvider(name = "skipAdminOptions")
```

Comment:
This data provider is very specific to a test case. Why not put it in the class where it's used?